cmd/ibazel/main.go

```diff
@@ -196,8 +197,24 @@ func main() {
 	handle(i, command, args)
 }
 
+func applyDefaultBazelArgs(bazelArgs []string) []string {
+	for _, bazelArg := range bazelArgs {
+		if strings.HasPrefix(bazelArg, "--isatty=") {
+			return bazelArgs
+		}
+	}
+	if (isTerminal()) {
```

Comment:
What are the consequences of turning this on? What does `--isatty` do to bazel? How does it behave differently? I'm having a hard time finding docs that describe the different behavior

Reply:
It's used to control the behavior of `--color=auto` and `--curses=auto`:

* [flag definition](https://github.com/bazelbuild/bazel/blob/49731660d8348d4ce243c4531bd5f3b14006f1df/src/main/java/com/google/devtools/build/lib/runtime/UiOptions.java#L173-L184)
* [flag references](https://github.com/bazelbuild/bazel/blob/49731660d8348d4ce243c4531bd5f3b14006f1df/src/main/java/com/google/devtools/build/lib/runtime/UiOptions.java#L284-L290)